hs/src/Reach/Eval/Core.hs

```diff
@@ -4951,7 +4967,6 @@ doToConsensus ks (ToConsensusRec {..}) = locAt slptc_at $ do
         void $ locSt st_pure $ evalApplyVals' req_rator [public bv, public $ SLV_Bytes at $ "non-network tokens distinct"]
```

Comment:
I was referring to this one... when you start to track a token, you should ensure that it is distinct from what you've done so far